uds/translator/service_definitions/input_output_control_by_identifier.py

```
@@ -0,0 +1,124 @@
+"""Translation for InputOutputControlByIdentifier (SID 0x2F) service."""
+
+__all__ = ["INPUT_OUTPUT_CONTROL_BY_IDENTIFIER",
+           "INPUT_OUTPUT_CONTROL_BY_IDENTIFIER_2020",
+           "INPUT_OUTPUT_CONTROL_BY_IDENTIFIER_2013"]
```

Ruff (0.14.4)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)